src/Illuminate/Support/helpers.php

```diff
@@ -523,3 +523,21 @@ function with($value, ?callable $callback = null)
         return is_null($callback) ? $value : $callback($value);
     }
 }
+
+if (! function_exists('to_english_numbers')) {
+    /**
+     * Converting Persian/Arabic digits to English
+     * @param  mixed  $value
+     * @return string
+     */
+    function to_english_numbers($value): string
```

Comment:
How would I know that this is specific to Persian/Arabic digits from the helper name alone? And these are not _English_, the _are_ Arabic numbers, just not [Eastern Arabic numerals](https://en.wikipedia.org/wiki/Eastern_Arabic_numerals), but [Western Arabic numerals](https://en.wikipedia.org/wiki/Arabic_numerals).

And what about other numbering systems?